src/Configuration/Parser/ContentTypesParser.php

```diff
@@ -124,6 +124,12 @@ protected function parseContentType($key, $contentType): array
             $contentType['singleton'] = false;
         }
 
+        if ($contentType['singleton']) {
+            $contentType['listing_records'] = 1;
+        } elseif (isset($contentType['listing_records']) === false) {
+            $contentType['listing_records'] = 6;
```

Comment:
shouldn't it default to what is set in main config?

Reply:
Yes, will change it